test/e2e/suites/conformance/conformance_test.go

```diff
@@ -57,7 +57,7 @@ var _ = ginkgo.Describe("[unmanaged] [conformance] tests", func() {
 	})
 	ginkgo.Measure(specName, func(b ginkgo.Benchmarker) {
 
-		name := fmt.Sprintf("cluster-%s", util.RandomString(6))
+		name := fmt.Sprintf("cluster-%s-%s", specName, util.RandomString(6))
```

Comment:
Thanks for the PR @ydarb.

When cluster name is set like this `ClusterName: fmt.Sprintf("%s-%s", specName, util.RandomString(6))`, template names are appended with `-cluster-template` as can be seen here:
https://gcsweb.k8s.io/gcs/kubernetes-jenkins/logs/periodic-cluster-api-e2e-main/1514973830395727872/artifacts/clusters/bootstrap/

We have an additional prefix cluster here. How about deleting it?

Reply:
My apologies. It was late when I made the updates 😄

Reply:
@sedefsavas Addressed your comment. Let me know if there is anything else that should be addressed.